common/djangoapps/student/helpers.py

```diff
@@ -297,7 +297,7 @@ def get_redirect_to(request):
     # get information about a user on edx.org. In any such case drop the parameter.
     if redirect_to:
         mime_type, _ = mimetypes.guess_type(redirect_to, strict=False)
-        if not http.is_safe_url(redirect_to):
+        if not http.is_safe_url(redirect_to, host=request.get_host()):
```

Comment:
It looks like the `host` kwarg is will be [removed in Django 2.1](https://github.com/django/django/blob/master/django/utils/http.py#L282). Can we use `allowed_hosts` instead?

Reply:
@douglashall Currently [`django==1.8.18`](https://github.com/edx/edx-platform/blob/master/requirements/edx/base.txt#L37) is being used in edx-platform and for this version [`is_safe_url`](https://github.com/django/django/blob/1.8.18/django/utils/http.py#L281) does not accept `allowed_hosts` argument.